channeld/channeld_wire.csv

```diff
@@ -104,6 +104,7 @@ msgdata,channeld_offer_htlc,cltv_expiry,u32,
 msgdata,channeld_offer_htlc,payment_hash,sha256,
 msgdata,channeld_offer_htlc,onion_routing_packet,u8,1366
 msgdata,channeld_offer_htlc,blinding,?pubkey,
+msgdata,channeld_offer_htlc,endorsed,?u8,
```

Comment:
Can this be ?bool Might make the rest clearer?

Reply:
much better yeah